packages/core/src/index.html

```diff
@@ -22,5 +22,20 @@
     <dso-alert level="danger">
       Dit is een testmelding
     </dso-alert>
+    <dso-badge level="primary">
```

Comment:
Als je ze dan toch toevoegt, doe dan alle varianten. Je mist nu `<dso-badge>` zonder `level`, een situatie die in met `.dso-badge` wel werd ondersteund: https://dso-toolkit.nl/12.0.0/components/detail/badge.html